src/javascript.rs

```diff
@@ -121,6 +126,39 @@ impl<'a> Generator<'a> {
         }
     }
 
+    fn import(
+        &mut self,
+        module: &'a Vec<String>,
+        as_name: &'a Option<String>,
+        unqualified: &'a Vec<UnqualifiedImport>,
+    ) -> Document<'a> {
+        let module_name = as_name
+            .as_ref()
+            .map(|n| n.as_str().to_doc())
+            .unwrap_or_else(|| Document::String(module.join("_")));
+        let path = Document::String(module.join("/")).surround("\"./", ".js\"");
+
+        let import_line = docvec!["import * as ", module_name.clone(), " from ", path, ";"];
+        match unqualified.len() {
+            0 => import_line,
+            _ => {
+                let matches = unqualified.into_iter().map(|i| match &i.as_name {
+                    Some(aliased_name) => docvec![i.name, ": ", aliased_name,],
+                    None => i.name.to_doc(),
+                });
+                docvec![
+                    import_line,
+                    line(),
+                    "const ",
+                    concat(Itertools::intersperse(matches, break_(",", ", "))).surround("{", "}"),
```

Comment:
I think this would be a good place to use an object helper to ensure that it formats correctly. At the moment this one renders like so:

```js
const {
one,
two,
three,
four,
five,
six,
} = module;
```
And one long import will cause every on in the file to be rendered on multiple lines.

Reply:
Ah I see this is in the other PR, thanks

Reply:
Uses helper

> Tests for the formatting of long lines would be good

Haven't added tests as all the long line formatting is checking in the custom records tests.